numcodecs/errors.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module defines custom exceptions that are raised in the `numcodecs` codebase.
+"""
+
+
+class UnknownCodecError(Exception):
```

Comment:
Should we make this inherit from `ValueError`?

Reply:
Since a `ValueError` was thrown originally, I simply assumed that was a well motivated choice.

My question was rather because changing the exception type could break code downstream. For example, in Polaris [we explicitly catch the `ValueError`](https://github.com/polaris-hub/polaris/blob/6a96b94bcc23a599baa07ff031676feb64ac7393/polaris/dataset/zarr/_utils.py#L39).

Reply:
if the error describes a codec that's a codec, but unknown, then it should be a `ValueError` IMO

Reply:
Ah, if it was originally a `ValueError`, then definitely 👍 to keep it as inheriting from `ValueError`